samples/unit-tests/chart/parallel-coordinates-updates/demo.js

```diff
@@ -144,4 +144,117 @@ QUnit.test('Update parallel coordinates plot', function (assert) {
         3,
         'After chart update parallelInfo.counter has correct value (#10081).'
     );
+
+    // Calculate yAxis extremes based on the range series points (#15752)
+    chart.series[0].setData([]);
+    chart.series[1].setData([]);
+
+    chart.update({
+        series: [{
+            type: 'arearange',
+            data: [
+                [23, 25],
+                [-1, -0.5],
+                [1.5, 4],
+                [1.5, 5],
+                [4, 5]
+            ]
+        }]
+    });
+
+    const points = chart.series[0].points;
+    let firstAxisMin = chart.yAxis[0].dataMin,
+        firstAxisMax = chart.yAxis[0].dataMax,
+        thirdAxisMin = chart.yAxis[2].dataMin,
+        thirdAxisMax = chart.yAxis[2].dataMax;
+
+    assert.strictEqual(
+        points[0].low,
+        firstAxisMin,
+        `The first yAxis' min should be equal to the first point's low value.`
+    );
+    assert.strictEqual(
+        points[0].high,
+        firstAxisMax,
+        `The first yAxis' max should be equal to the first point's high value.`
+    );
+
+    assert.strictEqual(
+        points[2].low,
+        thirdAxisMin,
+        `The third yAxis' min should be equal to the third point's low value.`
+    );
+    assert.strictEqual(
+        points[2].high,
+        thirdAxisMax,
+        `The third yAxis' max should be equal to the third point's high value.`
+    );
+
+    // Calculate plotHigh value based on each yAxis scale (#15752)
+    points.forEach(function (point, i) {
+        assert.strictEqual(
+            chart.yAxis[i].translate(point.high, 0, 1, 0, 1),
+            point.plotHigh,
+            `PlotHigh value should be calculated based on the point's yAxis.`
+        );
+    });
+
+    // Multiple series case (#15752)
+    chart.update({
+        series: [{
+            type: 'arearange',
+            data: [
+                [-20, -15],
+                [-10, -5],
+                [-10, -5]
+            ]
+        }, {
+            type: 'arearange',
+            data: [
+                [0, 1],
+                [0, 5],
+                [0, 8]
+            ]
+        }]
+    });
+
+    chart.addSeries({
+        type: 'arearange',
+        data: [
+            [10, 15],
+            [10, 15],
+            [-3, -1]
+        ]
+    });
```

Comment:
Some few inconsistencies:

1. You're using different data for single series tests, and new data for multiple series. Could you use one dataset? (the second preferable)
2. You're doing `chart.update()` and `chart.addSeries()` that redraw chart twice - a good pattern is to set a `redraw` flag to `false` for the first update, and redraw chart only once (better performance).
3. Why you're adding the second series using `update()`, but the third series using `addSeries`?
4. You have `type: 'arearange'` lines duplicated multiple times

Perhaps using only 2 series instead of 3 would be better (only one `addSeries` would be required). And you can set `chart: { type: 'arearange' }`.

Reply:
All right, thanks for the feedback 👍